.github/workflows/conda-python-tests.yaml

```
@@ -71,11 +71,7 @@ jobs:
           export MATRICES="
             pull-request:
               # amd64
-              - { ARCH: 'amd64', PY_VER: '3.9',  CUDA_VER: '11.4.3', LINUX_VER: 'centos7',     GPU: 'v100', DRIVER: 'earliest' }
               - { ARCH: 'amd64', PY_VER: '3.10', CUDA_VER: '11.8.0', LINUX_VER: 'ubuntu22.04', GPU: 'v100', DRIVER: 'latest'   }
```

We must test at least one conda job for each CUDA major version due to the importance of CUDA in our conda packages. Here, I expand this to two jobs and cover as much as possible across them.
```suggestion
              - { ARCH: 'arm64', PY_VER: '3.9', CUDA_VER: '11.8.0', LINUX_VER: 'rockylinux8', GPU: 'a100', DRIVER: 'latest'   }
              - { ARCH: 'amd64', PY_VER: '3.11', CUDA_VER: '12.2.2', LINUX_VER: 'ubuntu22.04', GPU: 'v100', DRIVER: 'latest'   }
```

---

I agree, that's a good point. How about on the wheel side? Do you think it's important to keep one job per CUDA version there? I'm torn, because I think it makes sense to have coverage and there are different problems that could happen for wheels, but I also don't know if it's worth the tradeoff right now. Wheels are definitely less likely to have problems with different CUDA versions than conda.

---

Resolved by discussion documented here: https://github.com/rapidsai/build-planning/issues/5#issuecomment-1973775147